### PR TITLE
Update DataAPI_change_log.md

### PR DESCRIPTION
**Pull request by PedroDebevere**

### CODE REVIEW

File: release-notes/DxMs/DataAPI_change_log.md [hunk 1]
```diff
@@ -7,6 +7,59 @@ uid: DataAPI_change_log
...
+
+The auto-generated connectors generated by the Data API no longer contain the copyright message. Instead, these just contain the following line, where the timestamp indicates the UTC timestamp of when the connector was generated:
+
+```csharp
+<!--This protocol is auto-generated by the DataMiner Data API at 2024-04-12T05:43:30.823Z.-->
+```
+
+### 29 April - Fix - DataAPI 1.2.0 - All active elements updated with same identifier
+
+When a user tries to push data to the Data API, and there are stopped elements with the specified identifier, the Data API will now update all active elements with that identifier.
```
Comment (MariekeGO) [inline release-notes/DxMs/DataAPI_change_log.md:24]: It's not really clear what the issue is that is fixed here. Could you rephrase this so this will be more clear for a user who may have encountered this issue?

File: release-notes/DxMs/DataAPI_change_log.md [hunk 2]
```diff
@@ -7,6 +7,59 @@ uid: DataAPI_change_log
...
+
+During startup of a node, the node will wait for 6 seconds before determining whether the node should be the leader. After the initial startup period, the node will determine whether it should be the leader based on the heartbeat messages received from the other nodes. A node will elect itself as leader if its node ID is larger than that of any of its peers.
+
+Before the leader announces itself as leader to the other nodes (via the heartbeat message), it will retrieve all auto-generated connectors from DataMiner and keep these in memory locally.
+
+The leader knows about all auto-generated connectors, and it will coordinate all updates to these auto-generated connectors by granting a lock to any node that wishes to perform an update on an auto-generated connector.
+
+The following sequence diagram shows the flow when, for example, a follower node receives JSON data that would result in an update of a connector.
```
Comment (MariekeGO) [inline release-notes/DxMs/DataAPI_change_log.md:36]: This diagram is missing. Should this be added, or should this sentence be removed/rephrased?

File: release-notes/DxMs/DataAPI_change_log.md (hunk 1 above)
Comment (PedroDebevere) [reply release-notes/DxMs/DataAPI_change_log.md]: updated description

File: release-notes/DxMs/DataAPI_change_log.md (hunk 2 above)
Comment (PedroDebevere) [reply release-notes/DxMs/DataAPI_change_log.md:36]: added diagram

**Review by MariekeGO — APPROVED** [review]
